src/components/DateInput/__tests__/DateInput.spec.js

```diff
@@ -56,7 +57,7 @@ describe('DateInput', () => {
 
     const textInput = getByLabelText(props.label);
 
-    await fireEvent.click(textInput);
+    await userEvent.click(textInput);
```

Comment:
HEADS UP!  We now need to `await` all `userEvent` functions in the latest update to the `user-event` library.

Reply:
:laugh-cry: :sigh:

Reply:
:oh-no: 🤦🏼‍♀️

Reply:
yeah I know, we're finally all getting used to _not_ awaiting `userEvent.click()` and now we need to remember to start doing it again...